src/include/mallocMC/mallocMC_utils.hpp

```diff
@@ -100,7 +101,7 @@ namespace CUDA
 
 
 #define warp_serial                                    \
-  for (uint32_richtig_huebsch __mask = __ballot(1),                    \
+  for (uint32_t __mask = __ballot(1),                    \
```

Comment:
wow, well spotted!
I will use the type directly, since this is the only line where this problem occurs.